daisy_workflows/image_import/import_image.sh

```diff
@@ -149,13 +169,11 @@ if ! out=$(gcloud -q compute instances attach-disk ${ME} --disk=${DISKNAME} --zo
 fi
 echo ${out}
 
-# Output the size of the persistent disks for debugging.
-# The '-i' parameter is for ascii; without it the output is
-# garbled in serial output.
-#
-# No df here since we're interested in /dev/sdc which doesn't
-# have a filesystem yet.
-lsblk -i
+if [[ ${SIZE_GB} -gt 10 ]]; then
```

Comment:
Can ${SIZE_GB} be less than 10 or it's rounded up to 10? If it is, this if statement is redundant and we could just do:
waitUntilDeviceAttached /dev/sdc "${SIZE_GB}"

Reply:
Yeah, it can be less than 10. The check has been used for awhile: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/dbe6534a73bef27454034d9eb274a31011319f3f/daisy_workflows/image_import/import_image.sh#L61